test/test_nn.py

```diff
@@ -3453,6 +3453,23 @@ def test_data_parallel_device_args(self):
         out = dp.data_parallel(l, i, device_ids=(cuda0, cuda1), output_device=cuda0)
         self.assertEqual(out, l(i))
 
+    @unittest.skipIf(not TEST_MULTIGPU, "multi-GPU not supported")
+    def test_data_parallel_with_embedding(self):
```

Comment:
Helpful to quote the bug in question, so it's easier for people to find out what bug this is fixing